src/scripts/scenes/TileVisual.ts

```diff
@@ -43,22 +47,24 @@ export class TileVisual extends Container {
    private readonly _notProducing: Sprite;
    private readonly _upgrade: Sprite;
    private readonly _level: BitmapText;
+   private readonly _storagePercentage: BitmapText;
    private readonly _constructionAnimation: Action;
    private readonly _upgradeAnimation: Action;
    private readonly _xy: Tile;
    private readonly _timeLeft: BitmapText;
+   private readonly _gs: GameState;
    private _lastFloaterShownAt = 0;
    private _floaterValue = 0;
 
    constructor(world: WorldScene, grid: IPointData) {
       super();
       this._world = world;
       this._grid = grid;
-      const gs = getGameState();
+      this._gs = getGameState();
```

Comment:
I don't think it's necessary to store a reference to the game state. Instead, when you need Game State, just call `getGameState()`